src/EnergyPlus/Autosizing/CoolingAirFlowSizing.cc

```diff
@@ -621,16 +621,16 @@ Real64 CoolingAirFlowSizer::size(EnergyPlusData &state, Real64 _originalValue, b
                         }
                     }
                 } else {
-                    if (state.dataSize->ZoneCoolingOnlyFan) {
+                    if (this->zoneEqSizing(this->curZoneEqNum).CoolingAirFlow) {
+                        this->autoSizedValue = this->zoneEqSizing(this->curZoneEqNum).CoolingAirVolFlow;
+                    } else if (state.dataSize->ZoneCoolingOnlyFan) {
```

Comment:
The order here makes a difference. Makes me wonder what the hierarchy should be.